week5_R_draft.ipynb

```diff
@@ -1,32 +1,8 @@
 {
```

Comment:
<p>I like your use of str because it really clears out stuff for the students and helps visualize what's going on</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>